<launch>
    <!-- Bring in global hardware parameters -->
    <rosparam command="load" file="$(find system_globals)/params/frames.yaml" />
    <rosparam command="load" file="$(find system_globals)/params/hardware_constants.yaml" />
    <rosparam command="load" file="$(find topic_messages)/params/topics.yaml" />
    <rosparam command="load" file="$(find walking_gait)/params/gait_state.yaml" />

    <node
        pkg="walking_gait"
        name="gait_state_control_node"
        type="gait_state_control_node.py"
        respawn="true"
        respawn_delay="1"
        output="screen"
    />
</launch>